dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/user/hibernate/HibernateUserSettingStore.java

```diff
@@ -101,12 +102,12 @@ public void deleteUserSetting(UserSetting userSetting) {
     getSession().delete(userSetting);
   }
 
-  @Override
-  public void removeUserSettings(User user) {
-    String hql = "delete from UserSetting us where us.user = :user";
-
-    getSession().createQuery(hql).setParameter("user", user).executeUpdate();
-  }
+  //  @Override
+  //  public void removeUserSettings(String username) {
+  //    String hql = "delete from UserSetting us where us.user.username = :username";
+  //
+  //    getSession().createQuery(hql).setParameter(username, username).executeUpdate();
+  //  }
```

Comment:
if not needed can we remove? if needed can a TODO be added so it's easily identifiable?